generators/keywords

```diff
@@ -28,13 +28,15 @@ except IndexError:
     print("Usage: ./generator <subdir>")
     sys.exit(1)
 
+test_dir = os.path.join(tests_dir, 'generated', tests_subdir)
+
+if not os.path.isdir(test_dir):
+    os.mkdir(test_dir)
+
 conf_file = os.path.join(conf_dir, "generators", "sv-keywords.list")
 with open(conf_file, "r") as f:
     for x in f:
         x = x[:-1]
-        test_file = os.path.join(tests_dir,
-                                 "generated",
-                                 tests_subdir,
-                                 "5.6.2--keyword-" + x + ".sv")
+        test_file = os.path.join(test_dir, "5.6.2--keyword-" + x + ".sv")
```

Comment:
Any reason why the keyword tests cannot use the same generator as the other tets with a simple template?

Reply:
I don't think that there are any, I will update the `keywords` generator to use this new approach